UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -291,6 +291,198 @@ <h4>Instructions</h4>
 			<section id="conformance-group">
 				<h3>Conformance</h3>
 				<p>This technique relates to <a href="https://www.w3.org/2021/09/UX-Guide-metadata-2.0/principles/#conformance-group">Conformance key information</a>.</p>
+				
+				<h4>Understanding the variables</h4>
+				<dl>
+					<dt><var>epub_accessibility_1_0</var></dt>
+					<dd>
+						<p>If true it indicates that either the <a href="https://ns.editeur.org/onix/en/196/02">code 02 of codelist 196</a> (EPUB Accessibility Specification 1.0 A) or the <a href="https://ns.editeur.org/onix/en/196/03">code 03 of codelist 196</a> (EPUB Accessibility Specification 1.0 AA) are present in the ONIX record, otherwise if false it means that the metadata is not present.</p>
+						<p>This means that the publication conforms with either the requirements of EPUB Accessibility Spec 1.0, both level A or level AA.</p>
```

Comment:
"both" seems to me should be removed as it implies Both A & AA even though you have "or" just sounds awkward.

Maybe "at" should be used instead?
This means that the publication conforms with either the requirements of EPUB Accessibility Spec 1.0, **at** level A or level AA.

Reply:
Fixed, thanks